yarn.lock

```diff
@@ -5233,13 +5233,6 @@
   resolved "https://registry.yarnpkg.com/@types/util-deprecate/-/util-deprecate-1.0.0.tgz#341d0815fe5a661b94e3ea738d182b4c359e3958"
   integrity sha512-I2vixiQ+mrmKxfdLNvaa766nulrMVDoUQiSQoNeTjFUNAt8klnMgDh3yy/bH/r275357q30ACOEUaxFOR8YVrA==
 
-"@types/webpack-bundle-analyzer@^3.9.0":
```

Comment:
Is this part the reason for the failure?
I couldn't see `@types/webpack-bundle-analyzer` in the stale lockfile diff of popper-core.

Reply:
I'm guessing it was an intermittent failure. We run so many builds that every now and then something gets corrupted. It seems to have fixed itself when I re-ran the job.